### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by jdozierezell** [wp-graphql/wp-graphql-acf#23]
Is this still under active development?

**Comment by jasonbahl** [comment]
@jdozierezell I’m not actively working on it at the moment. I do think it would be a super valuable plugin though so if I do get time to get back to it I will. It would also be cool to see someone in the community pick it up 😀. 

I don’t use ACF on any projects currently, so it’s hard for me to prioritize working on this right now.

**Comment by cr101** [comment]
I agree that this would be a super valuable plugin since [ACF for Gutenberg](https://www.advancedcustomfields.com/blog/the-state-of-acf-in-a-gutenberg-world/) will be released soon

**Comment by adamsoffer** [comment]
Hey @jasonbahl - out of curiosity what do you use for custom fields? I'd like to use wpgraphql but not sure how to handle custom fields.
